client/src/Components/CommonComponents/AutoCompleteTags/index.jsx

```diff
@@ -1,6 +1,116 @@
-import React from 'react';
-import UnderConstruction from '../../UnderConstruction';
+import React, { Component } from 'react';
+import { Spinner } from 'react-bootstrap';
+import { Typeahead } from 'react-bootstrap-typeahead';
+import PropTypes from 'prop-types';
+import 'react-bootstrap-typeahead/css/Typeahead.css';
+import { skills, offerType } from './staticData';
```

Comment:
maybe you should separate importing modules from importing local components and style